ci_framework/roles/dlrn_promote/molecule/default/converge.yml

```diff
@@ -18,5 +18,13 @@
   hosts: all
   vars:
     cifmw_dlrn_promote_hash: false
-  roles:
-    - role: "dlrn_promote"
+    cifmw_repo_setup_os_release: centos
+  tasks:
+    - name: Run repo-setup role
+      ansible.builtin.include_role:
+        name: repo_setup
```

Comment:
This one is ran in the bootstrap step, I assume it's safe to call this role multiple times, but is necessary?

Reply:
We run repo_setup with in dlrn_promote role. Since here we are testing the few pieces of the role. that's why I explicitly wanted to call it during molecule run.